sdk/eventhub/eventhubs-checkpointstore-blob/src/blobPartitionManager.ts

```diff
@@ -191,4 +191,18 @@ export class BlobPartitionManager implements PartitionManager {
       );
     }
   }
+
+  private static getBlobPrefix(params: { fullyQualifiedNamespace: string; eventHubName: string; consumerGroupName: string; partitionId?: string }): string {
+    // none of these are case-sensitive in eventhubs so we need to make sure we don't accidentally allow
```

Comment:
are we sure **none** of these are case sensitive?
From what I recall, we know consumer group is case insensitive for sure, am not sure about others
Did we test creating 2 event hubs with same name, different casing?

Reply:
I did :)

For all of these if you attempt to try different casing for any of these it just ends up being an 'update' to the lowercased version of the resource.

Reply:
We should let the other SDK teams know about that too so they do the same thing :)